### ISSUE TITLE
unable to start sandbox without aws credentials file in place

### ISSUE THREAD

**Issue by filmaj** [architect/utils#22]
If i rename my `~/.aws/credentials` file to something else, no matter what combination of various env vars I use (including omitting all the AWS env vars below), I cannot start sandbox, it blows up with:

```
➜ NODE_ENV=testing AWS_ACCESS_KEY_ID=foo AWS_SECRET_ACCESS_KEY=bar AWS_REGION=us-west-1 AWS_PROFILE=default arc sandbox
         app ⌁ arctonic
      region ⌁ us-west-2
     profile ⌁ default
     version ⌁ Architect CLI 6.0.20
         cwd ⌁ /Users/maj/src/arcandtonic.technology

✓ Found Architect manifest, starting up
✓ Project files hydrated into functions
{ CredentialsError: Missing credentials in config
    at Object.openSync (fs.js:438:3)
    at Object.readFileSync (fs.js:343:35)
    at Object.readFileSync (/Users/maj/src/arcandtonic.technology/node_modules/aws-sdk/lib/util.js:95:26)
    at IniLoader.parseFile (/Users/maj/src/arcandtonic.technology/node_modules/aws-sdk/lib/shared-ini/ini-loader.js:6:47)
    at IniLoader.loadFrom (/Users/maj/src/arcandtonic.technology/node_modules/aws-sdk/lib/shared-ini/ini-loader.js:56:30)
    at SharedIniFileCredentials.load (/Users/maj/src/arcandtonic.technology/node_modules/aws-sdk/lib/credentials/shared_ini_file_credentials.js:104:41)
    at SharedIniFileCredentials.coalesceRefresh (/Users/maj/src/arcandtonic.technology/node_modules/aws-sdk/lib/credentials.js:205:12)
    at SharedIniFileCredentials.refresh (/Users/maj/src/arcandtonic.technology/node_modules/aws-sdk/lib/credentials/shared_ini_file_credentials.js:185:10)
    at SharedIniFileCredentials.get (/Users/maj/src/arcandtonic.technology/node_modules/aws-sdk/lib/credentials.js:122:12)
    at getAsyncCredentials (/Users/maj/src/arcandtonic.technology/node_modules/aws-sdk/lib/config.js:361:24)
  message: 'Missing credentials in config',
  errno: -2,
  syscall: 'open',
  code: 'CredentialsError',
  path: '/Users/maj/.aws/credentials',
  time: 2019-08-03T15:39:32.828Z,
  originalError:
   { message: 'Could not load credentials from SharedIniFileCredentials',
     errno: -2,
     syscall: 'open',
     code: 'CredentialsError',
     path: '/Users/maj/.aws/credentials',
     time: 2019-08-03T15:39:32.828Z,
     originalError:
      { errno: -2,
        syscall: 'open',
        code: 'ENOENT',
        path: '/Users/maj/.aws/credentials',
        message:
         'ENOENT: no such file or directory, open \'/Users/maj/.aws/credentials\'' } } }
/Users/maj/src/arcandtonic.technology/node_modules/aws-sdk/lib/request.js:31
            throw err;
            ^

Error: Unable to list Dynamo tables
    at Response._tables (/Users/maj/src/arcandtonic.technology/node_modules/@architect/sandbox/src/db/create-table/_create-table.js:20:13)
    at Request.<anonymous> (/Users/maj/src/arcandtonic.technology/node_modules/aws-sdk/lib/request.js:364:18)
    at Request.callListeners (/Users/maj/src/arcandtonic.technology/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
    at Request.emit (/Users/maj/src/arcandtonic.technology/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
    at Request.emit (/Users/maj/src/arcandtonic.technology/node_modules/aws-sdk/lib/request.js:683:14)
    at Request.transition (/Users/maj/src/arcandtonic.technology/node_modules/aws-sdk/lib/request.js:22:10)
    at AcceptorStateMachine.runTo (/Users/maj/src/arcandtonic.technology/node_modules/aws-sdk/lib/state_machine.js:14:12)
    at /Users/maj/src/arcandtonic.technology/node_modules/aws-sdk/lib/state_machine.js:26:10
    at Request.<anonymous> (/Users/maj/src/arcandtonic.technology/node_modules/aws-sdk/lib/request.js:38:9)
    at Request.<anonymous> (/Users/maj/src/arcandtonic.technology/node_modules/aws-sdk/lib/request.js:685:12)
```

Are credentials required for sandbox running? If no, can we smooth over the experience here for users? At a minimum document the env vars needed in the README to make it run without actual AWS creds in place?

**Comment by ryanblock** [comment]
Fixed by `utils 1.4.x` (and `sandbox 1.5.2` due out later today)!

**Comment by filmaj** [comment]
Nice one! That was an epic one to dig through, thanks for resolving!
